src/components/AlertPopups/RejectBoxPopup/RejectBoxPopup.jsx

```diff
@@ -44,7 +44,8 @@ const RejectBoxPopup = ({
       sendEmail(
         boxHolderName,
         boxHolderEmail,
-        <RejectedBoxEmail boxHolderName={boxHolderName} rejectionReason={rejectionReason} />,
+        <AdminApprovalProcessEmail type="rejected" rejectionReason={rejectionReason} />,
+        'Find Your Anchor Launch Map - Update',
```

Comment:
nit - Could you make a constant for this subject line in utils since it's used for all aap emails? In case they would like to change the subject line in the future.